Build failure: prismlauncher on aarch64-darwin

### Nixpkgs version

- Unstable (25.05)

### Steps to reproduce

1. Use nixpkgs/nixos-unstable + commit b403bd0b59f14a166a43e7ffda334dac7ac17305 (from https://github.com/NixOS/nixpkgs/pull/398849) to fix another build error
2. `darwin-rebuild switch --flake ...`

see https://github.com/alexstaeding/.config/tree/broken-prismlauncher

### Can Hydra reproduce this build failure?

Hydra is currently rebuilding this package.

### Link to Hydra build job

_No response_

### Relevant log output

```console
Running phase: qtPreHook
@nix { "action": "setPhase", "phase": "qtPreHook" }
Running phase: unpackPhase
@nix { "action": "setPhase", "phase": "unpackPhase" }
unpacking source archive /nix/store/c18wx8pdwxlysjsf2mkspmja5gmv4w90-source
source root is source
Running phase: patchPhase
@nix { "action": "setPhase", "phase": "patchPhase" }
Running phase: updateAutotoolsGnuConfigScriptsPhase
@nix { "action": "setPhase", "phase": "updateAutotoolsGnuConfigScriptsPhase" }
Running phase: configurePhase
@nix { "action": "setPhase", "phase": "configurePhase" }
fixing cmake files...
cmake flags: -GNinja -DCMAKE_FIND_USE_SYSTEM_PACKAGE_REGISTRY=OFF -DCMAKE_FIND_USE_PACKAGE_REGISTRY=OFF -DCMAKE_EXPORT_NO_PACKAGE_REGISTRY=ON -DCMAKE_BUILD_TYPE=Release -DCMAKE_INSTALL_LOCALEDIR=/nix/store/4zlrbg1mip6ady8wfgfz52yn0phpjg9w-prismlauncher-unwrapped-9.4/share/locale -DCMAKE_INSTALL_LIBEXECDIR=/nix/store/4zlrbg1mip6ady8wfgfz52yn0phpjg9w-prismlauncher-unwrapped-9.4/libexec -DCMAKE_INSTALL_LIBDIR=/nix/store/4zlrbg1mip6ady8wfgfz52yn0phpjg9w-prismlauncher-unwrapped-9.4/lib -DCMAKE_INSTALL_DOCDIR=/nix/store/4zlrbg1mip6ady8wfgfz52yn0phpjg9w-prismlauncher-unwrapped-9.4/share/doc/Launcher -DCMAKE_INSTALL_INFODIR=/nix/store/4zlrbg1mip6ady8wfgfz52yn0phpjg9w-prismlauncher-unwrapped-9.4/share/info -DCMAKE_INSTALL_MANDIR=/nix/store/4zlrbg1mip6ady8wfgfz52yn0phpjg9w-prismlauncher-unwrapped-9.4/share/man -DCMAKE_INSTALL_INCLUDEDIR=/nix/store/4zlrbg1mip6ady8wfgfz52yn0phpjg9w-prismlauncher-unwrapped-9.4/include -DCMAKE_INSTALL_SBINDIR=/nix/store/4zlrbg1mip6ady8wfgfz52yn0phpjg9w-prismlauncher-unwrapped-9.4/sbin -DCMAKE_INSTALL_BINDIR=/nix/store/4zlrbg1mip6ady8wfgfz52yn0phpjg9w-prismlauncher-unwrapped-9.4/bin -DCMAKE_INSTALL_NAME_DIR=/nix/store/4zlrbg1mip6ady8wfgfz52yn0phpjg9w-prismlauncher-unwrapped-9.4/lib -DCMAKE_POLICY_DEFAULT_CMP0025=NEW -DCMAKE_FIND_FRAMEWORK=LAST -DCMAKE_STRIP=/nix/store/xpja3ad45p55aryvgng6g9grp8yivvsd-clang-wrapper-19.1.7/bin/strip -DCMAKE_RANLIB=/nix/store/xpja3ad45p55aryvgng6g9grp8yivvsd-clang-wrapper-19.1.7/bin/ranlib -DCMAKE_AR=/nix/store/xpja3ad45p55aryvgng6g9grp8yivvsd-clang-wrapper-19.1.7/bin/ar -DCMAKE_C_COMPILER=clang -DCMAKE_CXX_COMPILER=clang++ -DCMAKE_INSTALL_PREFIX=/nix/store/4zlrbg1mip6ady8wfgfz52yn0phpjg9w-prismlauncher-unwrapped-9.4 -DLauncher_BUILD_PLATFORM:STRING=nixpkgs -DINSTALL_BUNDLE:STRING=nodeps -DMACOSX_SPARKLE_UPDATE_FEED_URL:STRING=\'\' -DCMAKE_INSTALL_PREFIX:STRING=/nix/store/4zlrbg1mip6ady8wfgfz52yn0phpjg9w-prismlauncher-unwrapped-9.4/Applications/ -DKDE_INSTALL_EXECROOTDIR=/nix/store/4zlrbg1mip6ady8wfgfz52yn0phpjg9w-prismlauncher-unwrapped-9.4 -DKDE_INSTALL_BINDIR=/nix/store/4zlrbg1mip6ady8wfgfz52yn0phpjg9w-prismlauncher-unwrapped-9.4/bin -DKDE_INSTALL_SBINDIR=/nix/store/4zlrbg1mip6ady8wfgfz52yn0phpjg9w-prismlauncher-unwrapped-9.4/sbin -DKDE_INSTALL_LIBDIR=/nix/store/4zlrbg1mip6ady8wfgfz52yn0phpjg9w-prismlauncher-unwrapped-9.4/lib -DKDE_INSTALL_LIBEXECDIR=/nix/store/4zlrbg1mip6ady8wfgfz52yn0phpjg9w-prismlauncher-unwrapped-9.4/libexec -DKDE_INSTALL_CMAKEPACKAGEDIR=/nix/store/4zlrbg1mip6ady8wfgfz52yn0phpjg9w-prismlauncher-unwrapped-9.4/lib/cmake -DKDE_INSTALL_INCLUDEDIR=/nix/store/4zlrbg1mip6ady8wfgfz52yn0phpjg9w-prismlauncher-unwrapped-9.4/include -DKDE_INSTALL_LOCALSTATEDIR=/var -DKDE_INSTALL_DATAROOTDIR=/nix/store/4zlrbg1mip6ady8wfgfz52yn0phpjg9w-prismlauncher-unwrapped-9.4/share -DKDE_INSTALL_DATADIR=/nix/store/4zlrbg1mip6ady8wfgfz52yn0phpjg9w-prismlauncher-unwrapped-9.4/share -DKDE_INSTALL_DOCBUNDLEDIR=/nix/store/4zlrbg1mip6ady8wfgfz52yn0phpjg9w-prismlauncher-unwrapped-9.4/share/doc/HTML -DKDE_INSTALL_KCFGDIR=/nix/store/4zlrbg1mip6ady8wfgfz52yn0phpjg9w-prismlauncher-unwrapped-9.4/share/config.kcfg -DKDE_INSTALL_KCONFUPDATEDIR=/nix/store/4zlrbg1mip6ady8wfgfz52yn0phpjg9w-prismlauncher-unwrapped-9.4/share/kconf_update -DKDE_INSTALL_KSERVICES5DIR=/nix/store/4zlrbg1mip6ady8wfgfz52yn0phpjg9w-prismlauncher-unwrapped-9.4/share/kservices5 -DKDE_INSTALL_KSERVICETYPES5DIR=/nix/store/4zlrbg1mip6ady8wfgfz52yn0phpjg9w-prismlauncher-unwrapped-9.4/share/kservicetypes5 -DKDE_INSTALL_KXMLGUI5DIR=/nix/store/4zlrbg1mip6ady8wfgfz52yn0phpjg9w-prismlauncher-unwrapped-9.4/share/kxmlgui5 -DKDE_INSTALL_KNOTIFY5RCDIR=/nix/store/4zlrbg1mip6ady8wfgfz52yn0phpjg9w-prismlauncher-unwrapped-9.4/share/knotifications5 -DKDE_INSTALL_ICONDIR=/nix/store/4zlrbg1mip6ady8wfgfz52yn0phpjg9w-prismlauncher-unwrapped-9.4/share/icons -DKDE_INSTALL_LOCALEDIR=/nix/store/4zlrbg1mip6ady8wfgfz52yn0phpjg9w-prismlauncher-unwrapped-9.4/share/locale -DKDE_INSTALL_SOUNDDIR=/nix/store/4zlrbg1mip6ady8wfgfz52yn0phpjg9w-prismlauncher-unwrapped-9.4/share/sounds -DKDE_INSTALL_TEMPLATEDIR=/nix/store/4zlrbg1mip6ady8wfgfz52yn0phpjg9w-prismlauncher-unwrapped-9.4/share/templates -DKDE_INSTALL_WALLPAPERDIR=/nix/store/4zlrbg1mip6ady8wfgfz52yn0phpjg9w-prismlauncher-unwrapped-9.4/share/wallpapers -DKDE_INSTALL_APPDIR=/nix/store/4zlrbg1mip6ady8wfgfz52yn0phpjg9w-prismlauncher-unwrapped-9.4/share/applications -DKDE_INSTALL_DESKTOPDIR=/nix/store/4zlrbg1mip6ady8wfgfz52yn0phpjg9w-prismlauncher-unwrapped-9.4/share/desktop-directories -DKDE_INSTALL_MIMEDIR=/nix/store/4zlrbg1mip6ady8wfgfz52yn0phpjg9w-prismlauncher-unwrapped-9.4/share/mime/packages -DKDE_INSTALL_METAINFODIR=/nix/store/4zlrbg1mip6ady8wfgfz52yn0phpjg9w-prismlauncher-unwrapped-9.4/share/appdata -DKDE_INSTALL_MANDIR=/nix/store/4zlrbg1mip6ady8wfgfz52yn0phpjg9w-prismlauncher-unwrapped-9.4/share/man -DKDE_INSTALL_INFODIR=/nix/store/4zlrbg1mip6ady8wfgfz52yn0phpjg9w-prismlauncher-unwrapped-9.4/share/info -DKDE_INSTALL_DBUSDIR=/nix/store/4zlrbg1mip6ady8wfgfz52yn0phpjg9w-prismlauncher-unwrapped-9.4/share/dbus-1 -DKDE_INSTALL_DBUSINTERFACEDIR=/nix/store/4zlrbg1mip6ady8wfgfz52yn0phpjg9w-prismlauncher-unwrapped-9.4/share/dbus-1/interfaces -DKDE_INSTALL_DBUSSERVICEDIR=/nix/store/4zlrbg1mip6ady8wfgfz52yn0phpjg9w-prismlauncher-unwrapped-9.4/share/dbus-1/services -DKDE_INSTALL_DBUSSYSTEMSERVICEDIR=/nix/store/4zlrbg1mip6ady8wfgfz52yn0phpjg9w-prismlauncher-unwrapped-9.4/share/dbus-1/system-services -DKDE_INSTALL_SYSCONFDIR=/nix/store/4zlrbg1mip6ady8wfgfz52yn0phpjg9w-prismlauncher-unwrapped-9.4/etc -DKDE_INSTALL_CONFDIR=/nix/store/4zlrbg1mip6ady8wfgfz52yn0phpjg9w-prismlauncher-unwrapped-9.4/etc/xdg -DKDE_INSTALL_AUTOSTARTDIR=/nix/store/4zlrbg1mip6ady8wfgfz52yn0phpjg9w-prismlauncher-unwrapped-9.4/etc/xdg/autostart -DKDE_INSTALL_BUNDLEDIR=/nix/store/4zlrbg1mip6ady8wfgfz52yn0phpjg9w-prismlauncher-unwrapped-9.4/Applications/KDE -DKDE_INSTALL_QTPLUGINDIR=/nix/store/4zlrbg1mip6ady8wfgfz52yn0phpjg9w-prismlauncher-unwrapped-9.4/lib/qt-6/plugins -DKDE_INSTALL_PLUGINDIR=/nix/store/4zlrbg1mip6ady8wfgfz52yn0phpjg9w-prismlauncher-unwrapped-9.4/lib/qt-6/plugins -DKDE_INSTALL_QMLDIR=/nix/store/4zlrbg1mip6ady8wfgfz52yn0phpjg9w-prismlauncher-unwrapped-9.4/lib/qt-6/qml -DECM_DIR=/nix/store/85c199dk6f8r8lwyy7h5gg6mlr1mfi27-extra-cmake-modules-5.116.0/share/ECM/cmake
-- The C compiler identification is Clang 19.1.7
-- The CXX compiler identification is Clang 19.1.7
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /nix/store/xpja3ad45p55aryvgng6g9grp8yivvsd-clang-wrapper-19.1.7/bin/clang - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /nix/store/xpja3ad45p55aryvgng6g9grp8yivvsd-clang-wrapper-19.1.7/bin/clang++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
Error: Could not run sw_vers:
No such file or directory
Error: Could not run sw_vers:
No such file or directory
Error: Could not run sw_vers:
No such file or directory
-- Git commit: GITDIR-NOTFOUND
-- Git tag: GIT-NOTFOUND
-- Git refspec: GITDIR-NOTFOUND
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD - Success
-- Found Threads: TRUE
-- Performing Test HAVE_STDATOMIC
-- Performing Test HAVE_STDATOMIC - Success
-- Found WrapAtomic: TRUE
-- Found OpenGL: /nix/store/afinrggjfrk7q9pb7rv50l50cj0y0l63-apple-sdk-11.3/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/System/Library/Frameworks/OpenGL.framework
-- Found WrapOpenGL: TRUE
-- Found WrapVulkanHeaders: /nix/store/g6ikw4r78frsgfqypanrrmmnr5xwv43s-vulkan-headers-1.4.309.0/include
-- Found PkgConfig: /nix/store/hj7iydwvhkslzsvw82iilv8say0xv5n7-pkg-config-wrapper-0.29.2/bin/pkg-config (found version "0.29.2")
-- Checking for one of the modules 'scdoc'
[0mCMake Deprecation Warning at libraries/libnbtplusplus/CMakeLists.txt:1 (cmake_minimum_required):
  Compatibility with CMake < 3.10 will be removed from a future version of
  CMake.

  Update the VERSION argument <min> value.  Or, use the <min>...<max> syntax
  to tell CMake that the project requires at least <min> but has been updated
  to work with policies introduced by <max> or earlier.

[0m
-- Performing Test COMPILER_HAS_HIDDEN_VISIBILITY
-- Performing Test COMPILER_HAS_HIDDEN_VISIBILITY - Success
-- Performing Test COMPILER_HAS_HIDDEN_INLINE_VISIBILITY
-- Performing Test COMPILER_HAS_HIDDEN_INLINE_VISIBILITY - Success
-- Performing Test COMPILER_HAS_DEPRECATED_ATTR
-- Performing Test COMPILER_HAS_DEPRECATED_ATTR - Success
[0mCMake Deprecation Warning at libraries/launcher/CMakeLists.txt:1 (cmake_minimum_required):
  Compatibility with CMake < 3.10 will be removed from a future version of
  CMake.

  Update the VERSION argument <min> value.  Or, use the <min>...<max> syntax
  to tell CMake that the project requires at least <min> but has been updated
  to work with policies introduced by <max> or earlier.

[0m
-- Found Java: /nix/store/69zzx14rvpnwlzdf5rkvdwkak0nxi17z-zulu-ca-jdk-17.0.12/bin/java (found suitable version "17.0.12", minimum required is "1.7") found components: Development
[0mCMake Deprecation Warning at libraries/javacheck/CMakeLists.txt:1 (cmake_minimum_required):
  Compatibility with CMake < 3.10 will be removed from a future version of
  CMake.

  Update the VERSION argument <min> value.  Or, use the <min>...<max> syntax
  to tell CMake that the project requires at least <min> but has been updated
  to work with policies introduced by <max> or earlier.

[0m
-- Using system zlib
-- Using system QuaZip
[0mCMake Deprecation Warning at libraries/rainbow/CMakeLists.txt:1 (cmake_minimum_required):
  Compatibility with CMake < 3.10 will be removed from a future version of
  CMake.

  Update the VERSION argument <min> value.  Or, use the <min>...<max> syntax
  to tell CMake that the project requires at least <min> but has been updated
  to work with policies introduced by <max> or earlier.

[0m
[0mCMake Deprecation Warning at libraries/LocalPeer/CMakeLists.txt:1 (cmake_minimum_required):
  Compatibility with CMake < 3.10 will be removed from a future version of
  CMake.

  Update the VERSION argument <min> value.  Or, use the <min>...<max> syntax
  to tell CMake that the project requires at least <min> but has been updated
  to work with policies introduced by <max> or earlier.

[0m
-- Using system tomlplusplus
-- Using system cmark
[0mCMake Deprecation Warning at libraries/gamemode/CMakeLists.txt:1 (cmake_minimum_required):
  Compatibility with CMake < 3.10 will be removed from a future version of
  CMake.

  Update the VERSION argument <min> value.  Or, use the <min>...<max> syntax
  to tell CMake that the project requires at least <min> but has been updated
  to work with policies introduced by <max> or earlier.

[0m
[0mCMake Deprecation Warning at libraries/murmur2/CMakeLists.txt:1 (cmake_minimum_required):
  Compatibility with CMake < 3.10 will be removed from a future version of
  CMake.

  Update the VERSION argument <min> value.  Or, use the <min>...<max> syntax
  to tell CMake that the project requires at least <min> but has been updated
  to work with policies introduced by <max> or earlier.

[0m
-- Using system ghc_filesystem
[0mCMake Deprecation Warning at libraries/qdcss/CMakeLists.txt:1 (cmake_minimum_required):
  Compatibility with CMake < 3.10 will be removed from a future version of
  CMake.

  Update the VERSION argument <min> value.  Or, use the <min>...<max> syntax
  to tell CMake that the project requires at least <min> but has been updated
  to work with policies introduced by <max> or earlier.

[0m
-- Configuring done (3.3s)
-- Generating done (0.4s)
[33mCMake Warning:
  Manually-specified variables were not used by the project:

    CMAKE_EXPORT_NO_PACKAGE_REGISTRY
    CMAKE_POLICY_DEFAULT_CMP0025
    KDE_INSTALL_APPDIR
    KDE_INSTALL_AUTOSTARTDIR
    KDE_INSTALL_BINDIR
    KDE_INSTALL_BUNDLEDIR
    KDE_INSTALL_CMAKEPACKAGEDIR
    KDE_INSTALL_CONFDIR
    KDE_INSTALL_DATADIR
    KDE_INSTALL_DATAROOTDIR
    KDE_INSTALL_DBUSDIR
    KDE_INSTALL_DBUSINTERFACEDIR
    KDE_INSTALL_DBUSSERVICEDIR
    KDE_INSTALL_DBUSSYSTEMSERVICEDIR
    KDE_INSTALL_DESKTOPDIR
    KDE_INSTALL_DOCBUNDLEDIR
    KDE_INSTALL_EXECROOTDIR
    KDE_INSTALL_ICONDIR
    KDE_INSTALL_INCLUDEDIR
    KDE_INSTALL_INFODIR
    KDE_INSTALL_KCFGDIR
    KDE_INSTALL_KCONFUPDATEDIR
    KDE_INSTALL_KNOTIFY5RCDIR
    KDE_INSTALL_KSERVICES5DIR
    KDE_INSTALL_KSERVICETYPES5DIR
    KDE_INSTALL_KXMLGUI5DIR
    KDE_INSTALL_LIBDIR
    KDE_INSTALL_LIBEXECDIR
    KDE_INSTALL_LOCALEDIR
    KDE_INSTALL_LOCALSTATEDIR
    KDE_INSTALL_MANDIR
    KDE_INSTALL_METAINFODIR
    KDE_INSTALL_MIMEDIR
    KDE_INSTALL_PLUGINDIR
    KDE_INSTALL_QMLDIR
    KDE_INSTALL_QTPLUGINDIR
    KDE_INSTALL_SBINDIR
    KDE_INSTALL_SOUNDDIR
    KDE_INSTALL_SYSCONFDIR
    KDE_INSTALL_TEMPLATEDIR
    KDE_INSTALL_WALLPAPERDIR

[0m
-- Build files have been written to: /tmp/nix-build-prismlauncher-unwrapped-9.4.drv-0/source/build
cmake: enabled parallel building
cmake: enabled parallel installing
Running phase: buildPhase
@nix { "action": "setPhase", "phase": "buildPhase" }
build flags: -j12
[1/580] Automatic MOC for target BuildConfig
[2/580] Automatic MOC for target systeminfo
[3/580] Automatic MOC for target Launcher_rainbow
[4/580] Automatic MOC for target Launcher_murmur2
[5/580] Automatic MOC for target qdcss
[6/580] Automatic MOC for target nbt++
[7/580] Automatic MOC for target gamemode
[8/580] Building CXX object libraries/rainbow/CMakeFiles/Launcher_rainbow.dir/Launcher_rainbow_autogen/mocs_compilation.cpp.o
[9/580] Building CXX object buildconfig/CMakeFiles/BuildConfig.dir/BuildConfig_autogen/mocs_compilation.cpp.o
[10/580] Building CXX object libraries/systeminfo/CMakeFiles/systeminfo.dir/systeminfo_autogen/mocs_compilation.cpp.o
[11/580] Building CXX object libraries/libnbtplusplus/CMakeFiles/nbt++.dir/nbt++_autogen/mocs_compilation.cpp.o
[12/580] Building Java objects for JavaCheck.jar
warning: [options] bootstrap class path not set in conjunction with -source 7
warning: [options] source value 7 is obsolete and will be removed in a future release
warning: [options] target value 7 is obsolete and will be removed in a future release
warning: [options] To suppress warnings about obsolete options, use -Xlint:-options.
4 warnings
[13/580] Automatic MOC for target LocalPeer
[14/580] Building Java objects for NewLaunch.jar
warning: [options] bootstrap class path not set in conjunction with -source 7
warning: [options] source value 7 is obsolete and will be removed in a future release
warning: [options] target value 7 is obsolete and will be removed in a future release
warning: [options] To suppress warnings about obsolete options, use -Xlint:-options.
4 warnings
[15/580] Building CXX object libraries/libnbtplusplus/CMakeFiles/nbt++.dir/src/endian_str.cpp.o
[16/580] Building CXX object libraries/libnbtplusplus/CMakeFiles/nbt++.dir/src/tag_string.cpp.o
[17/580] Building CXX object libraries/libnbtplusplus/CMakeFiles/nbt++.dir/src/tag_compound.cpp.o
[18/580] Building CXX object libraries/libnbtplusplus/CMakeFiles/nbt++.dir/src/io/stream_writer.cpp.o
[19/580] Building CXX object libraries/libnbtplusplus/CMakeFiles/nbt++.dir/src/io/stream_reader.cpp.o
[20/580] Building CXX object libraries/libnbtplusplus/CMakeFiles/nbt++.dir/src/value.cpp.o
[21/580] Building CXX object libraries/libnbtplusplus/CMakeFiles/nbt++.dir/src/tag.cpp.o
[22/580] Building CXX object libraries/libnbtplusplus/CMakeFiles/nbt++.dir/src/value_initializer.cpp.o
[23/580] Building CXX object libraries/murmur2/CMakeFiles/Launcher_murmur2.dir/Launcher_murmur2_autogen/mocs_compilation.cpp.o
[24/580] Building CXX object libraries/libnbtplusplus/CMakeFiles/nbt++.dir/src/text/json_formatter.cpp.o
[25/580] Building CXX object buildconfig/CMakeFiles/BuildConfig.dir/BuildConfig.cpp.o
[26/580] Building CXX object libraries/qdcss/CMakeFiles/qdcss.dir/qdcss_autogen/mocs_compilation.cpp.o
[27/580] Building CXX object libraries/rainbow/CMakeFiles/Launcher_rainbow.dir/src/rainbow.cpp.o
[28/580] Generating CMakeFiles/NewLaunch.dir/java_class_filelist
[29/580] Linking CXX static library libraries/rainbow/libLauncher_rainbow.a
[30/580] Linking CXX static library buildconfig/libBuildConfig.a
[31/580] Building CXX object libraries/systeminfo/CMakeFiles/systeminfo.dir/src/sys_apple.cpp.o
[32/580] Generating CMakeFiles/JavaCheck.dir/java_class_filelist
[33/580] Building CXX object libraries/gamemode/CMakeFiles/gamemode.dir/gamemode_autogen/mocs_compilation.cpp.o
[34/580] Creating Java archive NewLaunch.jar
[35/580] Linking CXX static library libraries/gamemode/libgamemode.a
warning: /nix/store/xpja3ad45p55aryvgng6g9grp8yivvsd-clang-wrapper-19.1.7/bin/ranlib: archive library: libraries/gamemode/libgamemode.a the table of contents is empty (no object file members in the library define global symbols)
[36/580] Creating Java archive JavaCheck.jar
[37/580] Building CXX object libraries/libnbtplusplus/CMakeFiles/nbt++.dir/src/tag_list.cpp.o
[38/580] Linking CXX static library libraries/libnbtplusplus/libnbt++.a
[39/580] Building CXX object libraries/murmur2/CMakeFiles/Launcher_murmur2.dir/src/MurmurHash2.cpp.o
[40/580] Linking CXX static library libraries/murmur2/libLauncher_murmur2.a
[41/580] Building CXX object libraries/LocalPeer/CMakeFiles/LocalPeer.dir/LocalPeer_autogen/mocs_compilation.cpp.o
[42/580] Building CXX object libraries/systeminfo/CMakeFiles/systeminfo.dir/src/distroutils.cpp.o
[43/580] Linking CXX static library libraries/systeminfo/libsysteminfo.a
[44/580] Building CXX object libraries/LocalPeer/CMakeFiles/LocalPeer.dir/src/LockedFile.cpp.o
[45/580] Building Java objects for NewLaunchLegacy.jar
warning: [options] bootstrap class path not set in conjunction with -source 7
warning: [options] source value 7 is obsolete and will be removed in a future release
warning: [options] target value 7 is obsolete and will be removed in a future release
warning: [options] To suppress warnings about obsolete options, use -Xlint:-options.
legacy/org/prismlauncher/legacy/LegacyFrame.java:59: warning: [removal] Applet in java.applet has been deprecated and marked for removal
import java.applet.Applet;
                  ^
legacy/net/minecraft/Launcher.java:57: warning: [removal] Applet in java.applet has been deprecated and marked for removal
import java.applet.Applet;
                  ^
legacy/net/minecraft/Launcher.java:58: warning: [removal] AppletStub in java.applet has been deprecated and marked for removal
import java.applet.AppletStub;
                  ^
legacy/org/prismlauncher/legacy/LegacyLauncher.java:63: warning: [removal] Applet in java.applet has been deprecated and marked for removal
import java.applet.Applet;
                  ^
legacy/org/prismlauncher/legacy/LegacyFrame.java:82: warning: [removal] Applet in java.applet has been deprecated and marked for removal
    public LegacyFrame(String title, Applet applet) {
                                     ^
legacy/net/minecraft/Launcher.java:73: warning: [removal] Applet in java.applet has been deprecated and marked for removal
public final class Launcher extends Applet implements AppletStub {
                                    ^
legacy/net/minecraft/Launcher.java:73: warning: [removal] AppletStub in java.applet has been deprecated and marked for removal
public final class Launcher extends Applet implements AppletStub {
                                                      ^
legacy/net/minecraft/Launcher.java:78: warning: [removal] Applet in java.applet has been deprecated and marked for removal
    private Applet wrappedApplet;
            ^
legacy/net/minecraft/Launcher.java:82: warning: [removal] Applet in java.applet has been deprecated and marked for removal
    public Launcher(Applet applet) {
                    ^
legacy/net/minecraft/Launcher.java:86: warning: [removal] Applet in java.applet has been deprecated and marked for removal
    public Launcher(Applet applet, URL documentBase) {
                    ^
legacy/net/minecraft/Launcher.java:108: warning: [removal] Applet in java.applet has been deprecated and marked for removal
    public void replace(Applet applet) {
                        ^
legacy/org/prismlauncher/legacy/LegacyLauncher.java:126: warning: [removal] Applet in java.applet has been deprecated and marked for removal
    private static Applet createAppletClass(String clazz) throws Throwable {
                   ^
legacy/org/prismlauncher/legacy/LegacyLauncher.java:130: warning: [removal] Applet in java.applet has been deprecated and marked for removal
        return (Applet) appletConstructor.invoke();
                ^
17 warnings
[46/580] Building CXX object libraries/LocalPeer/CMakeFiles/LocalPeer.dir/src/LockedFile_unix.cpp.o
[47/580] Generating CMakeFiles/NewLaunchLegacy.dir/java_class_filelist
[48/580] Creating Java archive NewLaunchLegacy.jar
[49/580] Building CXX object libraries/LocalPeer/CMakeFiles/LocalPeer.dir/src/LocalPeer.cpp.o
[50/580] Linking CXX static library libraries/LocalPeer/libLocalPeer.a
[51/580] Building CXX object libraries/qdcss/CMakeFiles/qdcss.dir/src/qdcss.cpp.o
[52/580] Linking CXX static library libraries/qdcss/libqdcss.a
[53/580] Automatic MOC for target sys
[54/580] Building CXX object libraries/systeminfo/CMakeFiles/sys.dir/sys_autogen/mocs_compilation.cpp.o
[55/580] Building CXX object libraries/systeminfo/CMakeFiles/sys.dir/src/sys_test.cpp.o
[56/580] Linking CXX executable sys
ld: warning: dylib (/nix/store/alic6ng5gxrqhfnglcvczb5zvd5y6pjz-qtbase-6.9.0/lib/QtGui.framework/Versions/A/QtGui) was built for newer macOS version (12.0) than being linked (11.3)
ld: warning: dylib (/nix/store/alic6ng5gxrqhfnglcvczb5zvd5y6pjz-qtbase-6.9.0/lib/QtDBus.framework/Versions/A/QtDBus) was built for newer macOS version (12.0) than being linked (11.3)
ld: warning: dylib (/nix/store/alic6ng5gxrqhfnglcvczb5zvd5y6pjz-qtbase-6.9.0/lib/QtNetwork.framework/Versions/A/QtNetwork) was built for newer macOS version (12.0) than being linked (11.3)
ld: warning: dylib (/nix/store/yn4gfj30alcc2k07vyr2yvr1yj76cq7w-qt5compat-6.9.0/lib/QtCore5Compat.framework/Versions/A/QtCore5Compat) was built for newer macOS version (12.0) than being linked (11.3)
ld: warning: dylib (/nix/store/alic6ng5gxrqhfnglcvczb5zvd5y6pjz-qtbase-6.9.0/lib/QtCore.framework/Versions/A/QtCore) was built for newer macOS version (12.0) than being linked (11.3)
ld: warning: dylib (/nix/store/alic6ng5gxrqhfnglcvczb5zvd5y6pjz-qtbase-6.9.0/lib/QtTest.framework/Versions/A/QtTest) was built for newer macOS version (12.0) than being linked (11.3)
[57/580] Automatic MOC for target Launcher_logic
[58/580] Generating qrc_documents.cpp
[59/580] Generating qrc_breeze_dark.cpp
[60/580] Generating qrc_flat.cpp
[61/580] Generating qrc_iOS.cpp
[62/580] Generating qrc_pe_light.cpp
[63/580] Generating qrc_OSX.cpp
[64/580] Generating qrc_pe_dark.cpp
[65/580] Generating qrc_pe_blue.cpp
[66/580] Generating qrc_pe_colored.cpp
[67/580] Generating qrc_breeze_light.cpp
[68/580] Generating qrc_prismlauncher.cpp
[69/580] Generating ui_PasteWizardPage.h
[70/580] Generating ui_MainWindow.h
[71/580] Generating ui_LoginWizardPage.h
[72/580] Generating ui_JavaPage.h
[73/580] Generating ui_AccountListPage.h
[74/580] Generating ui_AutoJavaWizardPage.h
[75/580] Generating ui_APIPage.h
[76/580] Generating ui_LauncherPage.h
[77/580] Generating ui_ThemeWizardPage.h
[78/580] Generating ui_ProxyPage.h
[79/580] Generating ui_ExternalResourcesPage.h
[80/580] Generating ui_ServersPage.h
[81/580] Generating ui_NotesPage.h
[82/580] Generating ui_LogPage.h
[83/580] Generating ui_MinecraftPage.h
[84/580] Generating ui_ExternalToolsPage.h
[85/580] Generating ui_OtherLogsPage.h
[86/580] Generating ui_GameOptionsPage.h
[87/580] Generating ui_VersionPage.h
[88/580] Generating ui_InstanceSettingsPage.h
[89/580] Generating ui_ScreenshotsPage.h
[90/580] Generating ui_ManagedPackPage.h
[91/580] Generating ui_WorldListPage.h
[92/580] Generating ui_AtlOptionalModDialog.h
[93/580] Generating ui_AtlPage.h
[94/580] Generating ui_CustomPage.h
[95/580] Generating ui_ResourcePage.h
[96/580] Generating ui_FlamePage.h
[97/580] Generating ui_ImportFTBPage.h
[98/580] Generating ui_Page.h
[99/580] Generating ui_ModrinthPage.h
[100/580] Generating ui_OptionalModDialog.h
[101/580] Generating ui_TechnicPage.h
[102/580] Generating ui_ImportPage.h
[103/580] Generating ui_CustomCommands.h
[104/580] Generating ui_InstanceCardWidget.h
[105/580] Generating ui_InfoFrame.h
[106/580] Generating ui_EnvironmentVariables.h
[107/580] Generating ui_ModFilterWidget.h
[108/580] Generating ui_SubTaskProgressBar.h
[109/580] Generating ui_ThemeCustomizationWidget.h
[110/580] Generating ui_CopyInstanceDialog.h
[111/580] Generating ui_ProfileSetupDialog.h
[112/580] Generating ui_ProgressDialog.h
[113/580] Generating ui_ProfileSelectDialog.h
[114/580] Generating ui_NewInstanceDialog.h
[115/580] Generating ui_ExportInstanceDialog.h
[116/580] Generating ui_NewComponentDialog.h
[117/580] Generating ui_NewsDialog.h
[118/580] Generating ui_ExportPackDialog.h
[119/580] Generating ui_IconPickerDialog.h
[120/580] Generating ui_ImportResourceDialog.h
[121/580] Generating ui_ExportToModListDialog.h
[122/580] Generating ui_MSALoginDialog.h
[123/580] Generating ui_ReviewMessageBox.h
[124/580] Generating ui_AboutDialog.h
[125/580] Generating ui_OfflineLoginDialog.h
[126/580] Generating ui_BlockedModsDialog.h
[127/580] Generating ui_ChooseProviderDialog.h
[128/580] Generating ui_ScrollMessageBox.h
[129/580] Generating ui_SkinManageDialog.h
[130/580] Generating ui_UpdateAvailableDialog.h
[131/580] Automatic RCC for resources/flat_white/flat_white.qrc
[132/580] Generating qrc_backgrounds.cpp
[133/580] Generating qrc_multimc.cpp
[134/580] Building CXX object launcher/CMakeFiles/Launcher_logic.dir/MessageLevel.cpp.o
[135/580] Building CXX object launcher/CMakeFiles/Launcher_logic.dir/LoggedProcess.cpp.o
[136/580] Building CXX object launcher/CMakeFiles/Launcher_logic.dir/BaseVersionList.cpp.o
[137/580] Building CXX object launcher/CMakeFiles/Launcher_logic.dir/StringUtils.cpp.o
[138/580] Building CXX object launcher/CMakeFiles/Launcher_logic.dir/InstanceTask.cpp.o
[139/580] Building CXX object launcher/CMakeFiles/Launcher_logic.dir/InstanceCreationTask.cpp.o
[140/580] Building CXX object launcher/CMakeFiles/Launcher_logic.dir/Untar.cpp.o
[141/580] Building CXX object launcher/CMakeFiles/Launcher_logic.dir/InstanceCopyPrefs.cpp.o
[142/580] Building CXX object launcher/CMakeFiles/Launcher_logic.dir/BaseInstaller.cpp.o
[143/580] Building CXX object launcher/CMakeFiles/Launcher_logic.dir/GZip.cpp.o
[144/580] Building CXX object launcher/CMakeFiles/Launcher_logic.dir/BaseInstance.cpp.o
[145/580] Building CXX object launcher/CMakeFiles/Launcher_logic.dir/MMCZip.cpp.o
[146/580] Building CXX object launcher/CMakeFiles/Launcher_logic.dir/Filter.cpp.o
[147/580] Building CXX object launcher/CMakeFiles/Launcher_logic.dir/Commandline.cpp.o
[148/580] Building CXX object launcher/CMakeFiles/Launcher_logic.dir/Json.cpp.o
[149/580] Building CXX object launcher/CMakeFiles/Launcher_logic.dir/Version.cpp.o
[150/580] Building CXX object launcher/CMakeFiles/Launcher_logic.dir/Logging.cpp.o
[151/580] Building CXX object launcher/CMakeFiles/Launcher_logic.dir/RecursiveFileSystemWatcher.cpp.o
[152/580] Building CXX object launcher/CMakeFiles/Launcher_logic.dir/MMCTime.cpp.o
[153/580] Building CXX object launcher/CMakeFiles/Launcher_logic.dir/InstanceList.cpp.o
In file included from /tmp/nix-build-prismlauncher-unwrapped-9.4.drv-0/source/launcher/InstanceList.cpp:60:
/tmp/nix-build-prismlauncher-unwrapped-9.4.drv-0/source/launcher/NullInstance.h:74:10: warning: 'updateRuntimeContext' overrides a member function but is not marked 'override' [-Winconsistent-missing-override]
   74 |     void updateRuntimeContext()
      |          ^
/tmp/nix-build-prismlauncher-unwrapped-9.4.drv-0/source/launcher/BaseInstance.h:218:18: note: overridden virtual function is here
  218 |     virtual void updateRuntimeContext();
      |                  ^
1 warning generated.
[154/580] Building CXX object launcher/CMakeFiles/Launcher_logic.dir/ResourceDownloadTask.cpp.o
[155/580] Building CXX object launcher/CMakeFiles/Launcher_logic.dir/FileSystem.cpp.o
FAILED: launcher/CMakeFiles/Launcher_logic.dir/FileSystem.cpp.o 
/nix/store/xpja3ad45p55aryvgng6g9grp8yivvsd-clang-wrapper-19.1.7/bin/clang++ -DLAUNCHER_APPLICATION -DQT_CONCURRENT_LIB -DQT_CORE5COMPAT_LIB -DQT_CORE_LIB -DQT_GUI_LIB -DQT_NETWORKAUTH_LIB -DQT_NETWORK_LIB -DQT_NO_DEBUG -DQT_WIDGETS_LIB -DQT_XML_LIB -I/tmp/nix-build-prismlauncher-unwrapped-9.4.drv-0/source/build/launcher -I/tmp/nix-build-prismlauncher-unwrapped-9.4.drv-0/source/launcher -I/tmp/nix-build-prismlauncher-unwrapped-9.4.drv-0/source/build/launcher/Launcher_logic_autogen/include -I/tmp/nix-build-prismlauncher-unwrapped-9.4.drv-0/source/libraries/systeminfo/include -I/tmp/nix-build-prismlauncher-unwrapped-9.4.drv-0/source/libraries/murmur2 -I/tmp/nix-build-prismlauncher-unwrapped-9.4.drv-0/source/libraries/murmur2/src -I/tmp/nix-build-prismlauncher-unwrapped-9.4.drv-0/source/libraries/libnbtplusplus/include -I/tmp/nix-build-prismlauncher-unwrapped-9.4.drv-0/source/build/libraries/libnbtplusplus -I/tmp/nix-build-prismlauncher-unwrapped-9.4.drv-0/source/libraries/qdcss/include -I/tmp/nix-build-prismlauncher-unwrapped-9.4.drv-0/source/buildconfig -I/tmp/nix-build-prismlauncher-unwrapped-9.4.drv-0/source/libraries/LocalPeer/include -I/tmp/nix-build-prismlauncher-unwrapped-9.4.drv-0/source/libraries/rainbow/include -isystem /nix/store/alic6ng5gxrqhfnglcvczb5zvd5y6pjz-qtbase-6.9.0/lib/QtCore.framework/Headers -iframework /nix/store/alic6ng5gxrqhfnglcvczb5zvd5y6pjz-qtbase-6.9.0/lib -isystem /nix/store/alic6ng5gxrqhfnglcvczb5zvd5y6pjz-qtbase-6.9.0/mkspecs/macx-clang -isystem /nix/store/alic6ng5gxrqhfnglcvczb5zvd5y6pjz-qtbase-6.9.0/lib/QtGui.framework/Headers -isystem /nix/store/alic6ng5gxrqhfnglcvczb5zvd5y6pjz-qtbase-6.9.0/lib/QtNetwork.framework/Headers -isystem /nix/store/yn4gfj30alcc2k07vyr2yvr1yj76cq7w-qt5compat-6.9.0/lib/QtCore5Compat.framework/Headers -iframework /nix/store/yn4gfj30alcc2k07vyr2yvr1yj76cq7w-qt5compat-6.9.0/lib -isystem /nix/store/alic6ng5gxrqhfnglcvczb5zvd5y6pjz-qtbase-6.9.0/lib/QtWidgets.framework/Headers -isystem /nix/store/alic6ng5gxrqhfnglcvczb5zvd5y6pjz-qtbase-6.9.0/lib/QtXml.framework/Headers -isystem /nix/store/alic6ng5gxrqhfnglcvczb5zvd5y6pjz-qtbase-6.9.0/lib/QtConcurrent.framework/Headers -isystem /nix/store/jw2jdkyh58kd3giimr1w8c6jkzf36an2-qtnetworkauth-6.9.0/lib/QtNetworkAuth.framework/Headers -iframework /nix/store/jw2jdkyh58kd3giimr1w8c6jkzf36an2-qtnetworkauth-6.9.0/lib -isystem /nix/store/r50qpy1vdp0fx4r78la7yfiqsyrij4q1-quazip-1.5-dev/include/QuaZip-Qt6-1.5 -isystem /nix/store/r50qpy1vdp0fx4r78la7yfiqsyrij4q1-quazip-1.5-dev/include/QuaZip-Qt6-1.5/quazip -Wall -pedantic -fstack-protector-strong --param=ssp-buffer-size=4  -DQT_NO_DEPRECATED_WARNINGS=Y -DQT_DISABLE_DEPRECATED_BEFORE=0x050C00 -DTOML_ENABLE_FLOAT16=0 -O2 -D_FORTIFY_SOURCE=2 -O3 -DNDEBUG -std=gnu++17 -isysroot /nix/store/afinrggjfrk7q9pb7rv50l50cj0y0l63-apple-sdk-11.3/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk -DTOML_HEADER_ONLY=0 -DTOML_SHARED_LIB=1 -MD -MT launcher/CMakeFiles/Launcher_logic.dir/FileSystem.cpp.o -MF launcher/CMakeFiles/Launcher_logic.dir/FileSystem.cpp.o.d -o launcher/CMakeFiles/Launcher_logic.dir/FileSystem.cpp.o -c /tmp/nix-build-prismlauncher-unwrapped-9.4.drv-0/source/launcher/FileSystem.cpp
/tmp/nix-build-prismlauncher-unwrapped-9.4.drv-0/source/launcher/FileSystem.cpp:953:11: error: no viable conversion from 'QString' to 'QFile'
  953 |     QFile info = content.path() + "/Info.plist";
      |           ^      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/nix/store/alic6ng5gxrqhfnglcvczb5zvd5y6pjz-qtbase-6.9.0/lib/QtCore.framework/Headers/qfile.h:320:5: note: candidate constructor not viable: no known conversion from 'QString' to 'QFilePrivate &' for 1st argument
  320 |     QFile(QFilePrivate &dd, QObject *parent = nullptr);
      |     ^     ~~~~~~~~~~~~~~~~
/nix/store/alic6ng5gxrqhfnglcvczb5zvd5y6pjz-qtbase-6.9.0/lib/QtCore.framework/Headers/qfile.h:325:20: note: candidate constructor not viable: no known conversion from 'QString' to 'const QFile &' for 1st argument
  325 |     Q_DISABLE_COPY(QFile)
      |     ~~~~~~~~~~~~~~~^~~~~~
/nix/store/alic6ng5gxrqhfnglcvczb5zvd5y6pjz-qtbase-6.9.0/lib/QtCore.framework/Headers/qtclasshelpermacros.h:24:5: note: expanded from macro 'Q_DISABLE_COPY'
   24 |     Class(const Class &) = delete;\
      |     ^     ~~~~~~~~~~~~~
/nix/store/alic6ng5gxrqhfnglcvczb5zvd5y6pjz-qtbase-6.9.0/lib/QtCore.framework/Headers/qfile.h:101:26: note: explicit constructor is not a candidate
  101 |     QFILE_MAYBE_EXPLICIT QFile(const QString &name);
      |                          ^
/nix/store/alic6ng5gxrqhfnglcvczb5zvd5y6pjz-qtbase-6.9.0/lib/QtCore.framework/Headers/qfile.h:106:26: note: explicit constructor is not a candidate
  106 |     QFILE_MAYBE_EXPLICIT QFile(const T &name) : QFile(QtPrivate::fromFilesystemPath(name))
      |                          ^
/nix/store/alic6ng5gxrqhfnglcvczb5zvd5y6pjz-qtbase-6.9.0/lib/QtCore.framework/Headers/qfile.h:112:14: note: explicit constructor is not a candidate
  112 |     explicit QFile(QObject *parent);
      |              ^
/nix/store/alic6ng5gxrqhfnglcvczb5zvd5y6pjz-qtbase-6.9.0/lib/QtCore.framework/Headers/qstring.h:1063:23: note: candidate function
 1063 |     Q_IMPLICIT inline operator std::u16string_view() const noexcept;
      |                       ^
1 error generated.
[156/580] Building CXX object launcher/CMakeFiles/Launcher_logic.dir/InstanceCopyTask.cpp.o
In file included from /tmp/nix-build-prismlauncher-unwrapped-9.4.drv-0/source/launcher/InstanceCopyTask.cpp:6:
/tmp/nix-build-prismlauncher-unwrapped-9.4.drv-0/source/launcher/NullInstance.h:74:10: warning: 'updateRuntimeContext' overrides a member function but is not marked 'override' [-Winconsistent-missing-override]
   74 |     void updateRuntimeContext()
      |          ^
/tmp/nix-build-prismlauncher-unwrapped-9.4.drv-0/source/launcher/BaseInstance.h:218:18: note: overridden virtual function is here
  218 |     virtual void updateRuntimeContext();
      |                  ^
1 warning generated.
[157/580] Building CXX object launcher/CMakeFiles/Launcher_logic.dir/net/Download.cpp.o
[158/580] Building CXX object launcher/CMakeFiles/Launcher_logic.dir/net/Logging.cpp.o
[159/580] Building CXX object launcher/CMakeFiles/Launcher_logic.dir/InstanceImportTask.cpp.o
In file included from /tmp/nix-build-prismlauncher-unwrapped-9.4.drv-0/source/launcher/InstanceImportTask.cpp:42:
/tmp/nix-build-prismlauncher-unwrapped-9.4.drv-0/source/launcher/NullInstance.h:74:10: warning: 'updateRuntimeContext' overrides a member function but is not marked 'override' [-Winconsistent-missing-override]
   74 |     void updateRuntimeContext()
      |          ^
/tmp/nix-build-prismlauncher-unwrapped-9.4.drv-0/source/launcher/BaseInstance.h:218:18: note: overridden virtual function is here
  218 |     virtual void updateRuntimeContext();
      |                  ^
1 warning generated.
[160/580] Building CXX object launcher/CMakeFiles/Launcher_logic.dir/net/Upload.cpp.o
[161/580] Building CXX object launcher/CMakeFiles/Launcher_logic.dir/net/FileSink.cpp.o
/tmp/nix-build-prismlauncher-unwrapped-9.4.drv-0/source/launcher/net/FileSink.cpp:53:31: warning: passing no argument for the '...' parameter of a variadic macro is a C++20 extension [-Wc++20-extensions]
   53 |         qCCritical(taskNetLogC) << "Could not create folder for " + m_filename;
      |                               ^
/nix/store/alic6ng5gxrqhfnglcvczb5zvd5y6pjz-qtbase-6.9.0/lib/QtCore.framework/Headers/qloggingcategory.h:160:9: note: macro 'qCCritical' defined here
  160 | #define qCCritical(category, ...) QT_MESSAGE_LOGGER_COMMON(category, QtCriticalMsg).critical(__VA_ARGS__)
      |         ^
/tmp/nix-build-prismlauncher-unwrapped-9.4.drv-0/source/launcher/net/FileSink.cpp:60:31: warning: passing no argument for the '...' parameter of a variadic macro is a C++20 extension [-Wc++20-extensions]
   60 |         qCCritical(taskNetLogC) << "Could not open " + m_filename + " for writing";
      |                               ^
/nix/store/alic6ng5gxrqhfnglcvczb5zvd5y6pjz-qtbase-6.9.0/lib/QtCore.framework/Headers/qloggingcategory.h:160:9: note: macro 'qCCritical' defined here
  160 | #define qCCritical(category, ...) QT_MESSAGE_LOGGER_COMMON(category, QtCriticalMsg).critical(__VA_ARGS__)
      |         ^
/tmp/nix-build-prismlauncher-unwrapped-9.4.drv-0/source/launcher/net/FileSink.cpp:72:31: warning: passing no argument for the '...' parameter of a variadic macro is a C++20 extension [-Wc++20-extensions]
   72 |         qCCritical(taskNetLogC) << "Failed writing into " + m_filename;
      |                               ^
/nix/store/alic6ng5gxrqhfnglcvczb5zvd5y6pjz-qtbase-6.9.0/lib/QtCore.framework/Headers/qloggingcategory.h:160:9: note: macro 'qCCritical' defined here
  160 | #define qCCritical(category, ...) QT_MESSAGE_LOGGER_COMMON(category, QtCriticalMsg).critical(__VA_ARGS__)
      |         ^
/tmp/nix-build-prismlauncher-unwrapped-9.4.drv-0/source/launcher/net/FileSink.cpp:113:35: warning: passing no argument for the '...' parameter of a variadic macro is a C++20 extension [-Wc++20-extensions]
  113 |             qCCritical(taskNetLogC) << "Failed to commit changes to " << m_filename;
      |                                   ^
/nix/store/alic6ng5gxrqhfnglcvczb5zvd5y6pjz-qtbase-6.9.0/lib/QtCore.framework/Headers/qloggingcategory.h:160:9: note: macro 'qCCritical' defined here
  160 | #define qCCritical(category, ...) QT_MESSAGE_LOGGER_COMMON(category, QtCriticalMsg).critical(__VA_ARGS__)
      |         ^
4 warnings generated.
[162/580] Building CXX object launcher/CMakeFiles/Launcher_logic.dir/net/MetaCacheSink.cpp.o
/tmp/nix-build-prismlauncher-unwrapped-9.4.drv-0/source/launcher/net/MetaCacheSink.cpp:95:38: warning: passing no argument for the '...' parameter of a variadic macro is a C++20 extension [-Wc++20-extensions]
   95 |             qCDebug(taskMetaCacheLogC) << "Adding eternal cache entry:" << m_entry->getFullPath();
      |                                      ^
/nix/store/alic6ng5gxrqhfnglcvczb5zvd5y6pjz-qtbase-6.9.0/lib/QtCore.framework/Headers/qloggingcategory.h:157:9: note: macro 'qCDebug' defined here
  157 | #define qCDebug(category, ...) QT_MESSAGE_LOGGER_COMMON(category, QtDebugMsg).debug(__VA_ARGS__)
      |         ^
/tmp/nix-build-prismlauncher-unwrapped-9.4.drv-0/source/launcher/net/MetaCacheSink.cpp:99:38: warning: passing no argument for the '...' parameter of a variadic macro is a C++20 extension [-Wc++20-extensions]
   99 |             qCDebug(taskMetaCacheLogC) << "Parsing 'Cache-Control' header with" << cache_control_header;
      |                                      ^
/nix/store/alic6ng5gxrqhfnglcvczb5zvd5y6pjz-qtbase-6.9.0/lib/QtCore.framework/Headers/qloggingcategory.h:157:9: note: macro 'qCDebug' defined here
  157 | #define qCDebug(category, ...) QT_MESSAGE_LOGGER_COMMON(category, QtDebugMsg).debug(__VA_ARGS__)
      |         ^
/tmp/nix-build-prismlauncher-unwrapped-9.4.drv-0/source/launcher/net/MetaCacheSink.cpp:107:38: warning: passing no argument for the '...' parameter of a variadic macro is a C++20 extension [-Wc++20-extensions]
  107 |             qCDebug(taskMetaCacheLogC) << "Parsing 'Expires' header with" << expires_header;
      |                                      ^
/nix/store/alic6ng5gxrqhfnglcvczb5zvd5y6pjz-qtbase-6.9.0/lib/QtCore.framework/Headers/qloggingcategory.h:157:9: note: macro 'qCDebug' defined here
  157 | #define qCDebug(category, ...) QT_MESSAGE_LOGGER_COMMON(category, QtDebugMsg).debug(__VA_ARGS__)
      |         ^
/tmp/nix-build-prismlauncher-unwrapped-9.4.drv-0/source/launcher/net/MetaCacheSink.cpp:117:38: warning: passing no argument for the '...' parameter of a variadic macro is a C++20 extension [-Wc++20-extensions]
  117 |             qCDebug(taskMetaCacheLogC) << "Parsing 'Age' header with" << age_header;
      |                                      ^
/nix/store/alic6ng5gxrqhfnglcvczb5zvd5y6pjz-qtbase-6.9.0/lib/QtCore.framework/Headers/qloggingcategory.h:157:9: note: macro 'qCDebug' defined here
  157 | #define qCDebug(category, ...) QT_MESSAGE_LOGGER_COMMON(category, QtDebugMsg).debug(__VA_ARGS__)
      |         ^
4 warnings generated.
[163/580] Building CXX object launcher/CMakeFiles/Launcher_logic.dir/net/HttpMetaCache.cpp.o
/tmp/nix-build-prismlauncher-unwrapped-9.4.drv-0/source/launcher/net/HttpMetaCache.cpp:130:30: warning: passing no argument for the '...' parameter of a variadic macro is a C++20 extension [-Wc++20-extensions]
  130 |         qCWarning(taskNetLogC) << "[HttpMetaCache]"
      |                              ^
/nix/store/alic6ng5gxrqhfnglcvczb5zvd5y6pjz-qtbase-6.9.0/lib/QtCore.framework/Headers/qloggingcategory.h:159:9: note: macro 'qCWarning' defined here
  159 | #define qCWarning(category, ...) QT_MESSAGE_LOGGER_COMMON(category, QtWarningMsg).warning(__VA_ARGS__)
      |         ^
/tmp/nix-build-prismlauncher-unwrapped-9.4.drv-0/source/launcher/net/HttpMetaCache.cpp:144:41: warning: passing no argument for the '...' parameter of a variadic macro is a C++20 extension [-Wc++20-extensions]
  144 |         qCCritical(taskHttpMetaCacheLogC) << "Cannot add entry with unknown base: " << stale_entry->m_baseId.toLocal8Bit();
      |                                         ^
/nix/store/alic6ng5gxrqhfnglcvczb5zvd5y6pjz-qtbase-6.9.0/lib/QtCore.framework/Headers/qloggingcategory.h:160:9: note: macro 'qCCritical' defined here
  160 | #define qCCritical(category, ...) QT_MESSAGE_LOGGER_COMMON(category, QtCriticalMsg).critical(__VA_ARGS__)
      |         ^
/tmp/nix-build-prismlauncher-unwrapped-9.4.drv-0/source/launcher/net/HttpMetaCache.cpp:149:41: warning: passing no argument for the '...' parameter of a variadic macro is a C++20 extension [-Wc++20-extensions]
  149 |         qCCritical(taskHttpMetaCacheLogC) << "Cannot add stale entry: " << stale_entry->getFullPath().toLocal8Bit();
      |                                         ^
/nix/store/alic6ng5gxrqhfnglcvczb5zvd5y6pjz-qtbase-6.9.0/lib/QtCore.framework/Headers/qloggingcategory.h:160:9: note: macro 'qCCritical' defined here
  160 | #define qCCritical(category, ...) QT_MESSAGE_LOGGER_COMMON(category, QtCriticalMsg).critical(__VA_ARGS__)
      |         ^
/tmp/nix-build-prismlauncher-unwrapped-9.4.drv-0/source/launcher/net/HttpMetaCache.cpp:173:38: warning: passing no argument for the '...' parameter of a variadic macro is a C++20 extension [-Wc++20-extensions]
  173 |         qCDebug(taskHttpMetaCacheLogC) << "Evicting base" << base;
      |                                      ^
/nix/store/alic6ng5gxrqhfnglcvczb5zvd5y6pjz-qtbase-6.9.0/lib/QtCore.framework/Headers/qloggingcategory.h:157:9: note: macro 'qCDebug' defined here
  157 | #define qCDebug(category, ...) QT_MESSAGE_LOGGER_COMMON(category, QtDebugMsg).debug(__VA_ARGS__)
      |         ^
/tmp/nix-build-prismlauncher-unwrapped-9.4.drv-0/source/launcher/net/HttpMetaCache.cpp:176:48: warning: passing no argument for the '...' parameter of a variadic macro is a C++20 extension [-Wc++20-extensions]
  176 |                 qCWarning(taskHttpMetaCacheLogC) << "Unexpected missing cache entry" << entry->m_basePath;
      |                                                ^
/nix/store/alic6ng5gxrqhfnglcvczb5zvd5y6pjz-qtbase-6.9.0/lib/QtCore.framework/Headers/qloggingcategory.h:159:9: note: macro 'qCWarning' defined here
  159 | #define qCWarning(category, ...) QT_MESSAGE_LOGGER_COMMON(category, QtWarningMsg).warning(__VA_ARGS__)
      |         ^
/tmp/nix-build-prismlauncher-unwrapped-9.4.drv-0/source/launcher/net/HttpMetaCache.cpp:291:34: warning: passing no argument for the '...' parameter of a variadic macro is a C++20 extension [-Wc++20-extensions]
  291 |     qCDebug(taskHttpMetaCacheLogC) << "Saving metacache with" << m_entries.size() << "entries";
      |                                  ^
/nix/store/alic6ng5gxrqhfnglcvczb5zvd5y6pjz-qtbase-6.9.0/lib/QtCore.framework/Headers/qloggingcategory.h:157:9: note: macro 'qCDebug' defined here
  157 | #define qCDebug(category, ...) QT_MESSAGE_LOGGER_COMMON(category, QtDebugMsg).debug(__VA_ARGS__)
      |         ^
/tmp/nix-build-prismlauncher-unwrapped-9.4.drv-0/source/launcher/net/HttpMetaCache.cpp:326:40: warning: passing no argument for the '...' parameter of a variadic macro is a C++20 extension [-Wc++20-extensions]
  326 |         qCWarning(taskHttpMetaCacheLogC) << "Error writing cache:" << e.what();
      |                                        ^
/nix/store/alic6ng5gxrqhfnglcvczb5zvd5y6pjz-qtbase-6.9.0/lib/QtCore.framework/Headers/qloggingcategory.h:159:9: note: macro 'qCWarning' defined here
  159 | #define qCWarning(category, ...) QT_MESSAGE_LOGGER_COMMON(category, QtWarningMsg).warning(__VA_ARGS__)
      |         ^
7 warnings generated.
[164/580] Building CXX object launcher/CMakeFiles/Launcher_logic.dir/net/PasteUpload.cpp.o
/tmp/nix-build-prismlauncher-unwrapped-9.4.drv-0/source/launcher/net/PasteUpload.cpp:147:30: warning: passing no argument for the '...' parameter of a variadic macro is a C++20 extension [-Wc++20-extensions]
  147 |     qCCritical(taskUploadLogC) << getUid().toString() << "Network error: " << error;
      |                              ^
/nix/store/alic6ng5gxrqhfnglcvczb5zvd5y6pjz-qtbase-6.9.0/lib/QtCore.framework/Headers/qloggingcategory.h:160:9: note: macro 'qCCritical' defined here
  160 | #define qCCritical(category, ...) QT_MESSAGE_LOGGER_COMMON(category, QtCriticalMsg).critical(__VA_ARGS__)
      |         ^
/tmp/nix-build-prismlauncher-unwrapped-9.4.drv-0/source/launcher/net/PasteUpload.cpp:163:34: warning: passing no argument for the '...' parameter of a variadic macro is a C++20 extension [-Wc++20-extensions]
  163 |         qCCritical(taskUploadLogC) << getUid().toString() << m_uploadUrl << " returned unexpected status code " << statusCode
      |                                  ^
/nix/store/alic6ng5gxrqhfnglcvczb5zvd5y6pjz-qtbase-6.9.0/lib/QtCore.framework/Headers/qloggingcategory.h:160:9: note: macro 'qCCritical' defined here
  160 | #define qCCritical(category, ...) QT_MESSAGE_LOGGER_COMMON(category, QtCriticalMsg).critical(__VA_ARGS__)
      |         ^
/tmp/nix-build-prismlauncher-unwrapped-9.4.drv-0/source/launcher/net/PasteUpload.cpp:181:42: warning: passing no argument for the '...' parameter of a variadic macro is a C++20 extension [-Wc++20-extensions]
  181 |                 qCCritical(taskUploadLogC) << getUid().toString() << getUid().toString() << m_uploadUrl
      |                                          ^
/nix/store/alic6ng5gxrqhfnglcvczb5zvd5y6pjz-qtbase-6.9.0/lib/QtCore.framework/Headers/qloggingcategory.h:160:9: note: macro 'qCCritical' defined here
  160 | #define qCCritical(category, ...) QT_MESSAGE_LOGGER_COMMON(category, QtCriticalMsg).critical(__VA_ARGS__)
      |         ^
/tmp/nix-build-prismlauncher-unwrapped-9.4.drv-0/source/launcher/net/PasteUpload.cpp:197:46: warning: passing no argument for the '...' parameter of a variadic macro is a C++20 extension [-Wc++20-extensions]
  197 |                     qCCritical(taskUploadLogC) << getUid().toString() << m_uploadUrl << " returned error: " << error;
      |                                              ^
/nix/store/alic6ng5gxrqhfnglcvczb5zvd5y6pjz-qtbase-6.9.0/lib/QtCore.framework/Headers/qloggingcategory.h:160:9: note: macro 'qCCritical' defined here
  160 | #define qCCritical(category, ...) QT_MESSAGE_LOGGER_COMMON(category, QtCriticalMsg).critical(__VA_ARGS__)
      |         ^
/tmp/nix-build-prismlauncher-unwrapped-9.4.drv-0/source/launcher/net/PasteUpload.cpp:198:46: warning: passing no argument for the '...' parameter of a variadic macro is a C++20 extension [-Wc++20-extensions]
  198 |                     qCCritical(taskUploadLogC) << getUid().toString() << "Response body: " << data;
      |                                              ^
/nix/store/alic6ng5gxrqhfnglcvczb5zvd5y6pjz-qtbase-6.9.0/lib/QtCore.framework/Headers/qloggingcategory.h:160:9: note: macro 'qCCritical' defined here
  160 | #define qCCritical(category, ...) QT_MESSAGE_LOGGER_COMMON(category, QtCriticalMsg).critical(__VA_ARGS__)
      |         ^
/tmp/nix-build-prismlauncher-unwrapped-9.4.drv-0/source/launcher/net/PasteUpload.cpp:203:42: warning: passing no argument for the '...' parameter of a variadic macro is a C++20 extension [-Wc++20-extensions]
  203 |                 qCCritical(taskUploadLogC) << getUid().toString() << m_uploadUrl << " returned malformed response body: " << data;
      |                                          ^
/nix/store/alic6ng5gxrqhfnglcvczb5zvd5y6pjz-qtbase-6.9.0/lib/QtCore.framework/Headers/qloggingcategory.h:160:9: note: macro 'qCCritical' defined here
  160 | #define qCCritical(category, ...) QT_MESSAGE_LOGGER_COMMON(category, QtCriticalMsg).critical(__VA_ARGS__)
      |         ^
/tmp/nix-build-prismlauncher-unwrapped-9.4.drv-0/source/launcher/net/PasteUpload.cpp:220:46: warning: passing no argument for the '...' parameter of a variadic macro is a C++20 extension [-Wc++20-extensions]
  220 |                     qCCritical(taskUploadLogC) << getUid().toString() << m_uploadUrl << " returned error: " << error;
      |                                              ^
/nix/store/alic6ng5gxrqhfnglcvczb5zvd5y6pjz-qtbase-6.9.0/lib/QtCore.framework/Headers/qloggingcategory.h:160:9: note: macro 'qCCritical' defined here
  160 | #define qCCritical(category, ...) QT_MESSAGE_LOGGER_COMMON(category, QtCriticalMsg).critical(__VA_ARGS__)
      |         ^
/tmp/nix-build-prismlauncher-unwrapped-9.4.drv-0/source/launcher/net/PasteUpload.cpp:221:46: warning: passing no argument for the '...' parameter of a variadic macro is a C++20 extension [-Wc++20-extensions]
  221 |                     qCCritical(taskUploadLogC) << getUid().toString() << "Error message: " << message;
      |                                              ^
/nix/store/alic6ng5gxrqhfnglcvczb5zvd5y6pjz-qtbase-6.9.0/lib/QtCore.framework/Headers/qloggingcategory.h:160:9: note: macro 'qCCritical' defined here
  160 | #define qCCritical(category, ...) QT_MESSAGE_LOGGER_COMMON(category, QtCriticalMsg).critical(__VA_ARGS__)
      |         ^
/tmp/nix-build-prismlauncher-unwrapped-9.4.drv-0/source/launcher/net/PasteUpload.cpp:222:46: warning: passing no argument for the '...' parameter of a variadic macro is a C++20 extension [-Wc++20-extensions]
  222 |                     qCCritical(taskUploadLogC) << getUid().toString() << "Response body: " << data;
      |                                              ^
/nix/store/alic6ng5gxrqhfnglcvczb5zvd5y6pjz-qtbase-6.9.0/lib/QtCore.framework/Headers/qloggingcategory.h:160:9: note: macro 'qCCritical' defined here
  160 | #define qCCritical(category, ...) QT_MESSAGE_LOGGER_COMMON(category, QtCriticalMsg).critical(__VA_ARGS__)
      |         ^
/tmp/nix-build-prismlauncher-unwrapped-9.4.drv-0/source/launcher/net/PasteUpload.cpp:227:42: warning: passing no argument for the '...' parameter of a variadic macro is a C++20 extension [-Wc++20-extensions]
  227 |                 qCCritical(taskUploadLogC) << getUid().toString() << m_uploadUrl << " returned malformed response body: " << data;
      |                                          ^
/nix/store/alic6ng5gxrqhfnglcvczb5zvd5y6pjz-qtbase-6.9.0/lib/QtCore.framework/Headers/qloggingcategory.h:160:9: note: macro 'qCCritical' defined here
  160 | #define qCCritical(category, ...) QT_MESSAGE_LOGGER_COMMON(category, QtCriticalMsg).critical(__VA_ARGS__)
      |         ^
10 warnings generated.
[165/580] Building CXX object launcher/CMakeFiles/Launcher_logic.dir/net/NetJob.cpp.o
[166/580] Building CXX object launcher/CMakeFiles/Launcher_logic.dir/Launcher_logic_autogen/mocs_compilation.cpp.o
In file included from /tmp/nix-build-prismlauncher-unwrapped-9.4.drv-0/source/build/launcher/Launcher_logic_autogen/mocs_compilation.cpp:20:
In file included from /tmp/nix-build-prismlauncher-unwrapped-9.4.drv-0/source/build/launcher/Launcher_logic_autogen/EWIEGA46WW/moc_NullInstance.cpp:9:
/tmp/nix-build-prismlauncher-unwrapped-9.4.drv-0/source/build/launcher/Launcher_logic_autogen/EWIEGA46WW/../../../../launcher/NullInstance.h:74:10: warning: 'updateRuntimeContext' overrides a member function but is not marked 'override' [-Winconsistent-missing-override]
   74 |     void updateRuntimeContext()
      |          ^
/tmp/nix-build-prismlauncher-unwrapped-9.4.drv-0/source/launcher/BaseInstance.h:218:18: note: overridden virtual function is here
  218 |     virtual void updateRuntimeContext();
      |                  ^
1 warning generated.
ninja: build stopped: subcommand failed.
```

```
error: builder for '/nix/store/f5bdskfly4rfm393h6y887jxk135m7zn-prismlauncher-unwrapped-9.4.drv' failed with exit code 1;
       last 25 log lines:
       > /tmp/nix-build-prismlauncher-unwrapped-9.4.drv-0/source/launcher/net/PasteUpload.cpp:222:46: warning: passing no argument for the '...' parameter of a variadic macro is a C++20 extension [-Wc++20-extensions]
       >   222 |                     qCCritical(taskUploadLogC) << getUid().toString() << "Response body: " << data;
       >       |                                              ^
       > /nix/store/alic6ng5gxrqhfnglcvczb5zvd5y6pjz-qtbase-6.9.0/lib/QtCore.framework/Headers/qloggingcategory.h:160:9: note: macro 'qCCritical' defined here
       >   160 | #define qCCritical(category, ...) QT_MESSAGE_LOGGER_COMMON(category, QtCriticalMsg).critical(__VA_ARGS__)
       >       |         ^
       > /tmp/nix-build-prismlauncher-unwrapped-9.4.drv-0/source/launcher/net/PasteUpload.cpp:227:42: warning: passing no argument for the '...' parameter of a variadic macro is a C++20 extension [-Wc++20-extensions]
       >   227 |                 qCCritical(taskUploadLogC) << getUid().toString() << m_uploadUrl << " returned malformed response body: " << data;
       >       |                                          ^
       > /nix/store/alic6ng5gxrqhfnglcvczb5zvd5y6pjz-qtbase-6.9.0/lib/QtCore.framework/Headers/qloggingcategory.h:160:9: note: macro 'qCCritical' defined here
       >   160 | #define qCCritical(category, ...) QT_MESSAGE_LOGGER_COMMON(category, QtCriticalMsg).critical(__VA_ARGS__)
       >       |         ^
       > 10 warnings generated.
       > [165/580] Building CXX object launcher/CMakeFiles/Launcher_logic.dir/net/NetJob.cpp.o
       > [166/580] Building CXX object launcher/CMakeFiles/Launcher_logic.dir/Launcher_logic_autogen/mocs_compilation.cpp.o
       > In file included from /tmp/nix-build-prismlauncher-unwrapped-9.4.drv-0/source/build/launcher/Launcher_logic_autogen/mocs_compilation.cpp:20:
       > In file included from /tmp/nix-build-prismlauncher-unwrapped-9.4.drv-0/source/build/launcher/Launcher_logic_autogen/EWIEGA46WW/moc_NullInstance.cpp:9:
       > /tmp/nix-build-prismlauncher-unwrapped-9.4.drv-0/source/build/launcher/Launcher_logic_autogen/EWIEGA46WW/../../../../launcher/NullInstance.h:74:10: warning: 'updateRuntimeContext' overrides a member function but is not marked 'override' [-Winconsistent-missing-override]
       >    74 |     void updateRuntimeContext()
       >       |          ^
       > /tmp/nix-build-prismlauncher-unwrapped-9.4.drv-0/source/launcher/BaseInstance.h:218:18: note: overridden virtual function is here
       >   218 |     virtual void updateRuntimeContext();
       >       |                  ^
       > 1 warning generated.
       > ninja: build stopped: subcommand failed.
       For full logs, run:
         nix log /nix/store/f5bdskfly4rfm393h6y887jxk135m7zn-prismlauncher-unwrapped-9.4.drv
error: 1 dependencies of derivation '/nix/store/j9w9jx8sh399x49bd6wrwnkw8lgak7wh-prismlauncher-9.4.drv' failed to build
error: 1 dependencies of derivation '/nix/store/9i9z9p8caq1lzh2qrw56xb9wvlz4bgds-home-manager-applications.drv' failed to build
error: 1 dependencies of derivation '/nix/store/xvlbm9mnf99f7ig5pz6pddmxhrskypcx-home-manager-fonts.drv' failed to build
error: 1 dependencies of derivation '/nix/store/n59vs2rlgzvi6ddi3xfmvlrvhs3ynxrx-home-manager-path.drv' failed to build
error: 1 dependencies of derivation '/nix/store/nbrdfr25jlcyqa83c6hqn25wnbk375v1-home-manager-generation.drv' failed to build
error: 1 dependencies of derivation '/nix/store/h8zjgkan31x5sxajavar39lj799vmfab-activation-alex.drv' failed to build
error: 1 dependencies of derivation '/nix/store/31a156ikf1vmlq4wsbw0m94a5vf4f3yv-darwin-system-25.05.43975d7.drv' failed to build
```

### Additional context

Offending update: https://github.com/NixOS/nixpkgs/pull/396661

### System metadata

 - system: `"aarch64-darwin"`
 - host os: `Darwin 24.4.0, macOS 15.4.1`
 - multi-user?: `yes`
 - sandbox: `no`
 - version: `nix-env (Nix) 2.28.1`
 - channels(root): `""`
 - nixpkgs: `/nix/store/5ds20jm3x2s4z7wn3581r6lc9ybmh45b-source`

### Notify maintainers

@Scrumplex
@Minion3665 
@getchoo 

---

**Note for maintainers:** Please tag this issue in your pull request description. (i.e. `Resolves #ISSUE`.)


### I assert that this issue is relevant for Nixpkgs

- [x] I assert that this is a bug and not a support request.
- [x] I assert that this is not a [duplicate of an existing issue](https://github.com/NixOS/nixpkgs/issues?q=is%3Aopen+is%3Aissue+label%3A%220.kind%3A+build+failure%22). 
- [x] I assert that I have read the [NixOS Code of Conduct](https://github.com/NixOS/.github/blob/master/CODE_OF_CONDUCT.md) and agree to abide by it.

### Is this issue important to you?

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

--- a/pkgs/by-name/pr/prismlauncher-unwrapped/package.nix
+++ b/pkgs/by-name/pr/prismlauncher-unwrapped/package.nix
@@ -5,7 +5,7 @@
   cmake,
   cmark,
   extra-cmake-modules,
-  fetchpatch,
+  fetchpatch2,
   gamemode,
   ghc_filesystem,
   jdk17,
@@ -44,6 +44,16 @@ stdenv.mkDerivation (finalAttrs: {
     rm -rf source/libraries/libnbtplusplus
     ln -s ${libnbtplusplus} source/libraries/libnbtplusplus
   '';
+
+  patches = [
+    # https://github.com/PrismLauncher/PrismLauncher/pull/3622
+    # https://github.com/NixOS/nixpkgs/issues/400119
+    (fetchpatch2 {
+      name = "fix-qt6.9-compatibility.patch";
+      url = "https://github.com/PrismLauncher/PrismLauncher/commit/8bb9b168fb996df9209e1e34be854235eda3d42a.diff";
+      hash = "sha256-hOqWBrUrVUhMir2cfc10gu1i8prdNxefTyr7lH6KA2c=";
+    })
+  ];
 
   nativeBuildInputs = [
     cmake

